app/common/directives/mainsheet-container.html

```diff
@@ -0,0 +1,9 @@
+<div class="mainsheet" ng-show="showMainsheet" style="display: block;">
```

Comment:
Added `init active` and moved `opacity: 1` from `mainsheet-overlay` to `mainsheet`.